### DISCUSSION TITLE
wrist_roll_joint Diagnostic data hasn't been updated in a timeout problem

### DISCUSSION THREAD

**Discussion by jjnak** [HumanSupportRobot/QandA#80] [Q&A]
The robot (some node) stops with the following error after a while: 
/Joints/Joint (wrist_roll_joint)
Diagnostic data hasn't been updated in a moment.

I wonder if anyone has experienced the same issue. Your help would be greatly appreciated.

-Jun

**Comment by yutaro-ishida** [comment]
Hi Jun,

Please let me check, does this refer to an error that appears on the HSR head display?
```
The robot (some node) stops with the following error after a while:
/Joints/Joint (wrist_roll_joint)
```

Yuta

**Reply by jjnak** [reply]
Dear Yuta,

Thank you for your response. Yes that is the message error shown on the HSR head display. 

After I changed the battery of the emergency stop remote controller to the new ones, this no longer happenes. But occasionally, the robot suddenly stops saying "停止しました" without any message on the HSR head display. I'm not sure if this is an interference of the wifi signals with the emergency stop remote controller.

**Reply by yutaro-ishida** [reply]
Hi Jun,

When the robot is suddenly stoped, the green light on the remote emergency switch is tuned off?

**Reply by jjnak** [reply]
Dear Yuta,

So far, I was not looking at the light of the remote emergency switch. Next time, I'll pay attention to it when it happens.

Thank you,

-Jun

**Reply by yutaro-ishida** [reply]
Gotcha. 

If the light is turned off, most likely stopped by the remote emergency switch.
Since I haven't seen the actual device, I can't be certain whether it is a hardware failure of the remote emergency switch or radio interference.

FYI, in my experience, there was a case which seemed to be radio interference with Bluetooth earphones.

**Reply by jjnak** [reply]
Thank you for the information. It is good to know that there has been a potential case of radio interference with Bluetooth earphones.
